src/main/kotlin/de/cyface/collector/model/Attachment.kt

```diff
@@ -153,9 +153,9 @@ data class Attachment(
         val geoJson = toGeoJson(deviceMetaData, applicationMetaData, measurementMetaData, attachmentMetaData)
 
         val properties = geoJson.getJsonObject("properties")
-        properties.put(FormAttributes.DEVICE_ID.value, identifier.deviceIdentifier)
-        properties.put(FormAttributes.MEASUREMENT_ID.value, identifier.measurementIdentifier)
-        properties.put(FormAttributes.ATTACHMENT_ID.value, identifier.attachmentIdentifier)
+        properties.put(FormAttributes.DEVICE_ID.value, identifier.deviceIdentifier.toString())
+        properties.put(FormAttributes.MEASUREMENT_ID.value, identifier.measurementIdentifier.toString())
+        properties.put(FormAttributes.ATTACHMENT_ID.value, identifier.attachmentIdentifier.toString())
```

Comment:
In our other metadata database format (GridFS) we store all IDs as String.
I guess this still makes sense to avoid heavy data migration when we change the id format again.
We had this issue before, when we changed `userId` from a Mongo Id to a UUID (OAuth).

Or do you think we should store the attachment and measurement id as Integer (android specific SQLite id type)? If other platforms use another storage type and with that, another id type (e.g. UUID), we cannot support both sensor platforms with this metadata format.